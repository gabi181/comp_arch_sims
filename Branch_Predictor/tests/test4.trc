32 6 2 2 global_history local_tables not_using_share
0x3B0D00EC T 0x7F46B2A4
0x6820F050 N 0x03E44A8C
0x3B0D00EC N 0x7F46B2A4
0x3D712F74 N 0x41475E9C
0x594C792C T 0x2AB17E0C
0x002BE854 T 0x5A7DD53C
0x1F01C878 N 0x417744A8
0x3D712F74 T 0x41475E9C
0x002BE854 T 0x5A7DD53C
0x3B0D00EC T 0x7F46B2A4
0x3D712F74 N 0x41475E9C
0x3B0D00EC T 0x7F46B2A4
0x002BE854 T 0x5A7DD53C
