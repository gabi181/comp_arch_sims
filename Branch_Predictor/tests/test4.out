0x3b0d00ec N 0x3b0d00f0
0x6820f050 N 0x6820f054
0x3b0d00ec T 0x7f46b2a4
0x3d712f74 N 0x3d712f78
0x594c792c N 0x594c7930
0x2be854 N 0x2be858
0x1f01c878 N 0x1f01c87c
0x3d712f74 T 0x41475e9c
0x2be854 T 0x5a7dd53c
0x3b0d00ec T 0x7f46b2a4
0x3d712f74 T 0x41475e9c
0x3b0d00ec T 0x7f46b2a4
0x2be854 T 0x5a7dd53c
flush_num: 5, br_num: 13, size: 5158b
